src/client.ts

```diff
@@ -206,6 +208,42 @@ export class PureCloudWebrtcSdk extends WildEmitter {
     }
   }
 
+  /**
+   * Start a video conference. Not supported for guests.
+   *  `initialize()` must be called first.
```

Comment:
Do we have an error message that states this if a user tries to use this without calling `initialize()` first?

Reply:
If we run into issues we'll update it. This currently works just like everything else so it shouldn't be a surprise for anyone.